### PR TITLE
Script/uninstall

### PR DESCRIPTION
**Pull request by DinoMarlir**

### CODE REVIEW

File: packages/uninstall.sh [hunk 1]
```diff
@@ -1,2 +1,17 @@
-rm /usr/bin/laboratory
-rm -rf /usr/share/laboratory/
+#!/usr/bin/bash
+
+#delete servers
+#https://stackoverflow.com/questions/1885525/how-do-i-prompt-a-user-for-confirmation-in-bash-script
+
+read -p "Should all data and servers be deleted? (y/N) " -n 1 -r
+echo "~/laboratory"
```
Comment (mooziii) [inline packages/uninstall.sh:7]: Why do you print the folder here?

Comment (DinoMarlir) [reply packages/uninstall.sh:7]: So that the users know what is being deleted

Comment (mooziii) [reply packages/uninstall.sh:7]: well then add it to the prompt itself, i think printing something before reading the response may cause issues

Comment (DinoMarlir) [reply packages/uninstall.sh:7]: Okay

**Review by mooziii — APPROVED** [review]
works fine for me

**Pull request merged**